src/openxr/OpenXRHand.cpp

```diff
@@ -0,0 +1,157 @@
+/////////////////////////////////////////////////////////////////////////////////////
+// Our OpenXR hand GDNative object implemented through meshes
+
+#include <ARVRServer.hpp>
```

Comment:
Weird issue to be aware off. If you include openxr before the ARVRServer it gives a really weird nonsensical compile error on Windows.  Hence it is included first, and hence the empty line of clang-format gets upset about the order.